graphs/possible_bipartition.py

```diff
@@ -1,12 +1,56 @@
 # Can be used for BFS
-from collections import deque 
+from collections import deque
 
 def possible_bipartition(dislikes):
     """ Will return True or False if the given graph
         can be bipartitioned without neighboring nodes put
         into the same partition.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n+e) number of nodes in the graph and e is the numbner of edges
+        Space Complexity: O(n) storing 4 variables? Checked, G1, G2, queue. drop the constant
```

Comment:
✨ Yes, because `queue`, `checked`, `group1`, and `group2` each can hold at most n elements, where n is the number of nodes (or dogs) in `dislikes`. Nice work!